Description: don't push userdict in preparation for Type 1 fonts
 Apparently, Fontographer fonts require a writable dictionary
 on the dict stack.
 For some reason lost in mists of time,
 we pushed userdict for that purpose -
 the problem is, that leaves any random redefinition of operators
 in place when we interpret the font file.
 .
 Instead, we now push an empty, temporary dictionary
 for those Fontographer fonts.
 As we already explicitly push systemdict
 (before the writable dictionary)
 that ensures a consistent environment
 for the interpretation of Type 1 fonts.
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=d06af15
Author: Chris Liddell <chris.liddell@artifex.com>
Forwarded: yes
Bug: http://bugs.ghostscript.com/show_bug.cgi?id=699857
Last-Update: 2018-10-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Resource/Init/gs_type1.ps
+++ b/Resource/Init/gs_type1.ps
@@ -166,7 +166,7 @@
         % However, we can't use any of the other well-known dictionaries
         % (such as userdict), since the whole point of pushing systemdict
         % is to make sure that nothing important has been redefined.
-   /userdict .systemvar begin
+   32 dict begin
         % We can't just use `run', because we want to check for .PFB files.
    currentpacking
     { //false setpacking .loadfont1 //true setpacking }
